src/interface/src/styles/forms.scss

```diff
@@ -1,6 +1,11 @@
 @import "mixins";
 @import "colors";
 
+@mixin button-disabled {
```

Comment:
I'm not sure if this belongs here, since it's exclusively a form mixin, or in mixins